tests/behat/features/bootstrap/EventContext.php

```diff
@@ -541,6 +541,31 @@ public function createAnonymousEventEnrollees(TableNode $eventAnonymousEnrollees
     }
   }
 
+  /**
+   * Enable the calendar button with a given calendar.
+   *
+   * @Given ":calendar" calendar is enabled
```

Comment:
I propose rephrasing this slightly to help in discovery and also disambiguate from any internal calendar function we might have.

```suggestion
   * @Given add to calendar is enabled for :calendar
```

I've dropped the quotes around `:calendar` because `"iCal"` still works but it can also just be `iCal` (so it makes the quotes optional)

Reply:
Added